docs/dev_building_from_source.rst

```diff
@@ -0,0 +1,47 @@
+
+==============================
+Building libCellML from Source
+==============================
+
+This document covers building libCellML from source.  It is assumed that you already have the codebase downloaded and ready for building.  The variable LIBCELLML_SRC shall be used to refer to the directory containing the LICENSE file for libCellML.
```

Comment:
Hmm, I see your points. libCellML does work better as a project name, but I don't like creating an exception to the naming convention in code. So happy to go with the status quo.